src/test/java/nextstep/subway/unit/LineServiceMockTest.java

```diff
@@ -182,6 +205,10 @@ void deleteSectionWithNonLastStation() {
         return new Station(2L, "역삼역");
     }
 
+    private Station 삼성역() {
+        return new Station(3L, "삼성역");
+    }
```

Comment:
이렇게 계속 늘려가는 것보다는 인자를 받아서 생성해주는 걸로도 충분할 것 같아요 😅 
```suggestion
    private Station 역생성(Long id, String name) {
        return new Station(id, name);
    }
```

Reply:
아하 넵넵 감사합니다!